modules/exploits/linux/http/unraid_eval.rb

```diff
@@ -0,0 +1,95 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::PhpEXE
+
+  Rank = ExcellentRanking
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'        => 'Unraid 6.8.0 Remote Code Execution',
+        'Description' => %q(
+          This module exploits two vulnerabilities affecting Unraid 6.8.0.
```

Comment:
Earlier versions are also reportedly affected.

```suggestion
          This module exploits two vulnerabilities affecting Unraid 6.8.0 and prior.
```